src/main/java/frc/trigon/robot/subsystems/climber/placeholderclimber/PlaceholderClimberConstants.java

```diff
@@ -0,0 +1,142 @@
+            LEFT_MOTOR_VOLTAGE_SIGNAL = LEFT_MOTOR.getMotorVoltage(),
+            LEFT_MOTOR_CURRENT_SIGNAL = LEFT_MOTOR.getStatorCurrent();
+
+    static {
+        configureRightMotor();
+        configureLeftMotor();
+    }
+
+    private static void configureRightMotor() {
+        final TalonFXConfiguration config = new TalonFXConfiguration();
+
+        config.MotorOutput.Inverted = RIGHT_MOTOR_INVERTED_VALUE;
+        config.MotorOutput.NeutralMode = NEUTRAL_MODE_VALUE;
+        config.Audio.BeepOnBoot = false;
+        config.Audio.BeepOnConfig = false;
+
+        config.Slot0.kP = AVERAGE_P;
+        config.Slot0.kI = AVERAGE_I;
+        config.Slot0.kD = AVERAGE_D;
+        config.Slot0.kA = AVERAGE_KA;
+        config.Slot0.kG = AVERAGE_KG;
+        config.Slot0.kS = AVERAGE_KS;
+        config.Slot0.kV = AVERAGE_KV;
+
+        config.Slot1.kP = DIFFERENTIAL_P;
+        config.Slot1.kI = DIFFERENTIAL_I;
+        config.Slot1.kD = DIFFERENTIAL_D;
+        config.Slot1.kA = DIFFERENTIAL_KA;
+        config.Slot1.kG = DIFFERENTIAL_KG;
+        config.Slot1.kS = DIFFERENTIAL_KS;
+        config.Slot1.kV = DIFFERENTIAL_KV;
+
+        config.MotionMagic.MotionMagicCruiseVelocity = MOTION_MAGIC_VELOCITY;
+        config.MotionMagic.MotionMagicAcceleration = MOTION_MAGIC_ACCELERATION;
+        config.MotionMagic.MotionMagicJerk = MOTION_MAGIC_JERK;
+
+        RIGHT_MOTOR.getConfigurator().apply(config);
+
+        RIGHT_MOTOR_POSITION_SIGNAL.setUpdateFrequency(100);
+        RIGHT_MOTOR_VELOCITY_SIGNAL.setUpdateFrequency(100);
+        RIGHT_MOTOR_SETPOINT_SIGNAL.setUpdateFrequency(100);
+        RIGHT_MOTOR_VOLTAGE_SIGNAL.setUpdateFrequency(100);
+        RIGHT_MOTOR_CURRENT_SIGNAL.setUpdateFrequency(100);
+
+        RIGHT_MOTOR.optimizeBusUtilization();
+    }
+
+    private static void configureLeftMotor() {
+        final TalonFXConfiguration config = new TalonFXConfiguration();
+
+        config.MotorOutput.Inverted = LEFT_MOTOR_INVERTED_VALUE;
+        config.MotorOutput.NeutralMode = NEUTRAL_MODE_VALUE;
+        config.Audio.BeepOnBoot = false;
+        config.Audio.BeepOnConfig = false;
+
+        config.Slot0.kP = AVERAGE_P;
+        config.Slot0.kI = AVERAGE_I;
+        config.Slot0.kD = AVERAGE_D;
+        config.Slot0.kA = AVERAGE_KA;
+        config.Slot0.kG = AVERAGE_KG;
+        config.Slot0.kS = AVERAGE_KS;
+        config.Slot0.kV = AVERAGE_KV;
+
+        config.Slot1.kP = DIFFERENTIAL_P;
+        config.Slot1.kI = DIFFERENTIAL_I;
+        config.Slot1.kD = DIFFERENTIAL_D;
+        config.Slot1.kA = DIFFERENTIAL_KA;
+        config.Slot1.kG = DIFFERENTIAL_KG;
+        config.Slot1.kS = DIFFERENTIAL_KS;
+        config.Slot1.kV = DIFFERENTIAL_KV;
+
+        config.MotionMagic.MotionMagicCruiseVelocity = MOTION_MAGIC_VELOCITY;
+        config.MotionMagic.MotionMagicAcceleration = MOTION_MAGIC_ACCELERATION;
+        config.MotionMagic.MotionMagicJerk = MOTION_MAGIC_JERK;
```

Comment:
extract the creation of the config into a function that'll accept the inverted value since that's the only difference.